packages/core/util/index.ts

```diff
@@ -335,12 +335,8 @@ export function parseLocStringOneBased(
   }
   const prefix = location.slice(0, lastColonIdx)
   const suffix = location.slice(lastColonIdx + 1)
-  if (
-    isValidRefName(prefix, assemblyName) &&
-    isValidRefName(location, assemblyName)
-  ) {
-    throw new Error(`ambiguous location string: "${locString}"`)
-  } else if (isValidRefName(prefix, assemblyName)) {
+
+  if (isValidRefName(prefix, assemblyName)) {
```

Comment:
With this change, the refName parsing no longer implements the algorithm described in Appendix A of https://samtools.github.io/hts-specs/SAMv1.pdf. For example, if you have the following refNames in your assembly:

```
ctgABC
ctgABC:3389
ctgXYZ
ctgXYZ:5819
ctgXYZ:9842
```

then trying to parse the locString `ctgXYZ:5819` is ambiguous and should be an error, since it could refer to the whole contig `ctgXYZ:5819` or the contig `ctgXYZ` at position 5819.

This is a contrived example, but since colons are allowed in refNames, it is possible.

Reply:
Gotcha. I guess to avoid this in the particular case of sorting the spreadsheet, maybe the locstring column could store extra information so as to avoid stepping to the parseLocString function..